templates/containers/index.html

```diff
@@ -2,215 +2,228 @@
-        <div class="col-3 col-medium-3">
-          <h3 class="p-heading--5">Base images</h3>
-        </div>
-        <div class="col-6 col-medium-3">
-          <p>Ubuntu base images provide a solid foundation for your applications, offering a stable and secure environment, built from the world’s most popular cloud operating system. Canonical provides commercial support for all Ubuntu-based container images.</p>
-          <p>
-            <a href="/support">Explore contract support&nbsp;&rsaquo;</a>
-          </p>
-        </div>
+
+  <div class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Explore our offering</h2>
       </div>
-      <hr class="p-rule"/>
-      <div class="row p-section--shallow">
-        <div class="col-3 col-medium-3">
-          <h3 class="p-heading--5">Application images</h3>
-        </div>
-        <div class="col-6 col-medium-3">
-          <p>Canonical's LTS Docker images come with a long-term support and stability commitment. Pre-built and pre-configured application, runtime, and base images supported for up to 10 years with security patches.</p>
-          <p>
-            <a href="https://hub.docker.com/u/ubuntu">Explore our OCI container images&nbsp;&rsaquo;</a>
-          </p>
-        </div>
+      <div class="col">
+        <p>
+          Ubuntu containers are designed for modern software deployments. Our solutions are tailored to diverse needs, from individual developers to large enterprises that need long term supported, secure and compliant containers.
+        </p>
       </div>
-      <hr class="p-rule"/>
-      <div class="row">
-        <div class="col-3 col-medium-3">
-          <h3 class="p-heading--5">Chiselled Ubuntu images</h3>
+    </div>
+    <div class="row">
+      <div class="col-start-large-4 col-9 col-medium-6">
+        <hr class="p-rule" />
+        <div class="row p-section--shallow">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">Container Build Service</h3>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              Have Canonical's expert team custom build secure, minimal and optimised containers for your application. This service is ideal for enterprise that need a trusted partner to create and maintain containers that offer:
+            </p>
+            <ul class="p-list--divided">
+              <li class="p-list__item is-ticked">Minimal attack surface, coupled with an optimised and consistent structure</li>
+              <li class="p-list__item is-ticked">12 years of security support</li>
+              <li class="p-list__item is-ticked">
+                Compliance with the most stringent security management, auditing and reporting frameworks
+              </li>
+            </ul>
+            <p>
+              <a href="/containers/contact-us" class="js-invoke-modal">Explore the Container Build Service&nbsp;&rsaquo;</a>
```

Comment:
In the doc this leads to `containers/chiselled#get-in-touch`

Reply:
It's the same form 🙂